src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services9_compiled.php

```diff
@@ -299,7 +299,7 @@ protected function getMethodCall1Service()
 
         $instance->setBar($this->get('foo'));
         $instance->setBar(NULL);
-        $instance->setBar(($this->get("foo")->foo() . (($this->hasParameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
+        $instance->setBar(($this->get('foo')->foo() . (($this->hasParameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
```

Comment:
You changed an instance of double quotes to single quotes on this line, but left three other instances untouched. Why not change them all to single quotes?